src/udev/meson.build

```diff
@@ -53,6 +53,28 @@ if conf.get('HAVE_ACL') == 1
         libudevd_core_sources += ['udev-builtin-uaccess.c']
 endif
 
+if conf.get('HAVE_STAP') == 1
+        probes_h = custom_target(
+                'probes.h',
+                input : 'probes.d',
+                output : 'probes.h',
+                command : ['dtrace', '-C', '-h', '-s', '@INPUT@', '-o', '@OUTPUT@'])
```

Comment:
Maybe check if meson generators can be used for this? (We didn't use them so far because in the past they didn't work too well, but meson has progressed quite a bit, so maybe it'd be worth trying again.)

Reply:
I'd need to use two generators where output of one is input of the other. That unfortunately still doesn't work with current meson,

https://github.com/mesonbuild/meson/issues/1141
https://github.com/mesonbuild/meson/issues/5401